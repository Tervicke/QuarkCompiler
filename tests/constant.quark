const integer = 42
print(integer)

const str = "helloworld"
print(str)

const d = 56.900
print(d)
