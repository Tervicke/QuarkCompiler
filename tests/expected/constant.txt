42
helloworld
56.9
